CompilerSource/compiler/compile.cpp

```diff
@@ -615,16 +618,24 @@ wto << "namespace enigma_user {\nstring shader_get_name(int i) {\n switch (i) {\
 //  int makeres = better_system("cd ","/MacOS/");
 //  int makeres = better_system(MAKE_location,"MacOS");
 
-  // Pick a file and flush it
-  const string redirfile = (makedir + "enigma_compile.log");
-  fclose(fopen(redirfile.c_str(),"wb"));
+  string flags = "";
+
+  if (redirect_make) {
+    // Pick a file and flush it
+    const string redirfile = (makedir + "enigma_compile.log");
+    fclose(fopen(redirfile.c_str(),"wb"));
```

Comment:
This line's technically dangerous, but I guess it can stay, since it's been there forever....